### PR TITLE
VZ-7805: Update access/secret key creation details in backup doc

### PR DESCRIPTION
**Pull request by theabmitra**

### CODE REVIEW

File: content/en/docs/uninstall/backup/prerequisites.md [hunk 1]
```diff
@@ -66,7 +66,12 @@ Next, meet the following prerequisite requirements for both `velero` and `ranche
 
 - Object store region information.
 
-- A signing key, which is required to authenticate with the Amazon S3 compatible object store. Follow these steps to create a [Customer Secret Key](https://docs.oracle.com/en-us/iaas/Content/Identity/Tasks/managingcredentials.htm#Working2).
+- Object store signing key. 
+  - A signing key, which is required to authenticate with the Amazon S3 compatible object store. 
+     - This special signing key is an Access Key/Secret Key pair. 
+     - Oracle provides the Access Key that is associated with your Console user login. 
+     - You or your administrator generates the Customer Secret key to pair with the Access Key.
+  - Follow these steps to create a [Customer Secret Key](https://docs.oracle.com/en-us/iaas/Content/Identity/Tasks/managingcredentials.htm#create-secret-key).
```
Comment (MelwynPaul) [inline content/en/docs/uninstall/backup/prerequisites.md:74]: Updates:

- This special signing key is an Access Key/Secret Key pair. > This is an Access key or a Secret Key pair.
- To create a Customer Secret key, see [Customer Secret key](https://docs.oracle.com/en-us/iaas/Content/Identity/Tasks/managingcredentials.htm#create-secret-key).

**Review by MelwynPaul — COMMENTED** [review]
Looks good. 
Added a couple of comments.

**Review by MelwynPaul — APPROVED** [review]
Looks good. 
Thank you.

File: content/en/docs/uninstall/backup/prerequisites.md [hunk 2]
```diff
@@ -66,7 +66,12 @@ Next, meet the following prerequisite requirements for both `velero` and `ranche
 
 - Object store region information.
 
-- A signing key, which is required to authenticate with the Amazon S3 compatible object store. Follow these steps to create a [Customer Secret Key](https://docs.oracle.com/en-us/iaas/Content/Identity/Tasks/managingcredentials.htm#Working2).
+- Object store signing key. 
+  - A signing key, which is required to authenticate with the Amazon S3 compatible object store. 
+     - This is an Access key or a Secret Key pair.
```
Comment (rosemarymarano) [inline content/en/docs/uninstall/backup/prerequisites.md:71]: Access key -> Access Key (As much as I don't like the use of capitalization in the referred to doc, https://docs.oracle.com/en-us/iaas/Content/Identity/Tasks/managingcredentials.htm#Working2, we should keep in line with it). And, more importantly, consistent in our own docs (In the next bulleted item Access Key, both words capitalized.

File: content/en/docs/uninstall/backup/prerequisites.md [hunk 3]
```diff
@@ -66,7 +66,12 @@ Next, meet the following prerequisite requirements for both `velero` and `ranche
 
 - Object store region information.
 
-- A signing key, which is required to authenticate with the Amazon S3 compatible object store. Follow these steps to create a [Customer Secret Key](https://docs.oracle.com/en-us/iaas/Content/Identity/Tasks/managingcredentials.htm#Working2).
+- Object store signing key. 
+  - A signing key, which is required to authenticate with the Amazon S3 compatible object store. 
+     - This is an Access key or a Secret Key pair. 
+     - Oracle provides the Access Key that is associated with your Console user login. 
+     - You or your administrator generates the Customer Secret key to pair with the Access Key.
+  - To create a Customer Secret key, see [Customer Secret Key](https://docs.oracle.com/en-us/iaas/Content/Identity/Tasks/managingcredentials.htm#create-secret-key).
```
Comment (rosemarymarano) [inline content/en/docs/uninstall/backup/prerequisites.md:74]: [Customer Secret Key] -> [Customer Secret key]

**Review by rosemarymarano — COMMENTED** [review]
Few corrections needed.

**Pull request merged**